ansible/roles/delayed_job/tasks/main.yml

```diff
@@ -1,45 +1,28 @@
 ---
   - set_fact: delayed_job_installed="true"
 
-  - name: Monit Stop All
-    shell: monit stop all
+  - name: Install systemd delayed_job script
     become: true
-    ignore_errors: yes
-
-  - name: Wait for monit to stop
-    shell: monit status | grep Monitored | wc -l | awk '{print $1 $2}'
-    register: monit_stopped
-    retries: 10
-    until: monit_stopped.stdout == "0"
-    delay: 10
-    become: true
-
-  - name: Install delayed_job monit script
+    vars:
+      job_queue: "{{ item }}"
+      loop_index: "{{ loop_index }}"
     template:
-      src: delayed-job-monit-rc
-      dest: /etc/monit/conf.d/delayed_job_{{project_name}}_{{rails_env}}
+      src: delayed-job-systemd.service
+      dest: /etc/systemd/system/delayed_job_{{ item }}{{ loop_index }}.service
+    loop: "{{ job_queues }}"
+    loop_control:
+      index_var: loop_index
+      loop_var: item
+
+  - name: Enable systemd delayed_job service
     become: true
+    systemd:
+      name: "delayed_job_{{ item }}{{ loop_index }}"
+      daemon_reload: true
+      enabled: yes
+      state: started
+    loop: "{{ job_queues }}"
+    loop_control:
+      loop_var: item
+      index_var: loop_index
 
-  - name: Remove old upstart files
```

Comment:
Do we want a "remove old monit config" thing?  Or do we just reprovision old servers?

Reply:
Oh, I kind of thought we had to reprovision old servers. Is that not the case? By reprovision, do you mean just create entirely new instances? 

We should really terraform all the things, we have several oldbuntu servers out there on various projects, but also it might be good to do a thing to remove old monit

Reply:
I think you should make new servers really, but if you want to snowflake it then go delete monit manually or something.